cmd/kubectl-gadget/utils/trace.go

```diff
@@ -465,13 +467,30 @@ func getTraceWatcher(traceID string) (watch.Interface, error) {
 func waitForCondition(traceID string, conditionFunction func(*gadgetv1alpha1.Trace) bool) (*gadgetv1alpha1.TraceList, error) {
 	var returnedTraces gadgetv1alpha1.TraceList
 
-	tracesNumber := 0
+	traceList, err := getTraceListFromID(traceID)
+	if err != nil {
+		return nil, err
+	}
+
+	tracesNumber := len(traceList.Items)
 	watchedTracesNumber := 0
 
+	minResourceVersion := 0
```

Comment:
Can you add a comment explaining what this resourceVersion means in this context and why you need to look for the min value?